datacamp/category.json

```diff
@@ -0,0 +1,4 @@
+{
+  "description": "Learn-by-doing in the comfort of your own browser. DataCamp teaches you Python for data science at your own pace! Interactive coding challenges and fun quizzes related to the videos can be found on `DataCamp's Website <https://www.datacamp.com/>`__.",
```

Comment:
Thanks you for your contribution! However, our philosophy of keeping everything here free somewhat conflicts with advertising courses on our site. Can you update your description to something less promotional? Thank you.